### PR TITLE
service skeleton prepared

### PR DESCRIPTION
**Pull request by pzabolotniy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,20 @@
...
+```bash
+go install -v google.golang.org/protobuf/cmd/protoc-gen-go@latest
+go install -v google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
+
+protoc --go_out=. --go-grpc_out=. proto/users.proto
+```
+## Launch
+
+```bash
+docker-compose up --build -d --remove-orphans
```
Comment (Sugar-pack) [inline README.md:14]: Maybe its better to add this commands to Makefile

File: internal/config/config.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+// AppConfig is a container for application config
+type AppConfig struct {
+	API *API `mapstructure:"api"`
+	Db  *DB  `mapstructure:"db"`
+}
+
+// GetAppConfig returns *Config
+func GetAppConfig() (*AppConfig, error) {
+	viper.SetConfigName("config")
```
Comment (Sugar-pack) [inline internal/config/config.go:32]: Please add an explanation (comment/ readme) that this config file is hardcoded

File: README.md (hunk 1 above)
Comment (pzabolotniy) [reply README.md:14]: i don't remember Makefile syntax. It will take some time to implement it. Maybe later

File: internal/config/config.go (hunk 2 above)
Comment (pzabolotniy) [reply internal/config/config.go]: added both inline comment and README description

**Pull request merged**